EIC-EA/EIC-community-D8
# Feature/eicnet 571 group feature

### Done:
- Created an oec_group_features module, which provides a plugin manager for Group Features
- Created a Discussions GroupFeature plugin, with a custom Group permission to access the Discussions overview
- Created an additional field to the Group entity type to store the selected features

### How to test:
- Create a group
- Select/unselect the Discussions feature from the Group edit form
- Check that the menu item is created/enabled/disabled based on the feature state (enabled/disabled)
- Same for the "Access Discussions overview" and "Entity: Add content item entities (discussion)"

### To do:
- [ ] Apply the permissions to roles based on the Group visibility: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-748
- [ ] Create missing plugins to handle the other features: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-749

### Known issues:
- [ ] The /discussions view is not accessible at all because of this issue: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-746
